fetch-in-react-practice/client/components/app.jsx

```diff
@@ -93,9 +93,10 @@ export default class App extends React.Component {
       .then(res => res.json())
       .then(foundTodo => {
         const origTodos = this.state.todos;
+        const newTodos = origTodos;
         foundTodo.isCompleted = newStatus.isCompleted;
-        origTodos[findTodoIndex] = foundTodo;
-        this.setState({ todos: origTodos });
+        newTodos[findTodoIndex] = foundTodo;
+        this.setState({ todos: newTodos });
```

Comment:
To make a legit copy you can use `.slice()` or `...` (spread)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#description